site/source/docs/api_reference/html5.h.rst

```diff
@@ -106,7 +106,7 @@ When the event occurs the callback is invoked with the relevant event "type" (fo
 
 .. _callback-handler-return-em_bool-html5-api:
 
-Callback handlers that return an :c:data:`bool` may specify ``true`` to signal that the handler *consumed* the event (this suppresses the default action for that event by calling its ``.preventDefault();`` member). Returning ``false`` indicates that the event was not consumed — the default browser event action is carried out and the event is allowed to pass on/bubble up as normal.
+Callback handlers that return a ``bool`` may specify ``true`` to signal that the handler *consumed* the event (this suppresses the default action for that event by calling its ``.preventDefault();`` member). Returning ``false`` indicates that the event was not consumed — the default browser event action is carried out and the event is allowed to pass on/bubble up as normal.
```

Comment:
What happened to `:c:data:`?

Reply:
With ``` :c:data:`bool` ``` I get this error ```emscripten/site/source/docs/api_reference/html5.h.rst:109: WARNING: Unparseable C cross-reference: 'bool'
Invalid C declaration: Expected identifier, got user-defined keyword: bool. Remove it from c_extra_keywords to allow it as identifier.
Currently c_extra_keywords is ['alignas', 'alignof', 'bool', 'complex', 'imaginary', 'noreturn', 'static_assert', 'thread_local']. [error at 4]
  bool
  ----^```

I think it makes more sense to remove the cross reference rather than changing `c_extra_keywords` in `conf.py`